test/strategy-heat-capacitor.test.js

```diff
@@ -0,0 +1,81 @@
+const { DateTime } = require("luxon");
+const expect = require("expect");
+const { calculate_schedule, calculate_opportunities, find_best_buy_sell_pattern, calculate_value_dictlist, remove_low_buysell_pairs } = require("../src/strategy-heat-capacitor-functions");
+const converted_prices = require("./data/converted-prices.json");
+const decreasing_end_prices = require("./data/tibber-decreasing2-24h.json");
+const { cloneDeep } = require("lodash");
+
+//User input
+const time_heat_1c = 60
+const time_cool_1c = 40
+const max_temp_adjustment = 1
+const min_saving_NOK_kWh = 0.1
+
+describe("Test heat capacitor strategy functions", () => {
+  before(function() {
+    prices =  converted_prices.priceData.slice(0, 1).map((p) => p.value);
+    decreasing_24h_prices =  decreasing_end_prices.priceData.slice(0, 1).map((p) => p.value);
+    start_date =  DateTime.fromISO(converted_prices.priceData[0].start);
+    buy_pattern= Array(Math.round(time_heat_1c*max_temp_adjustment*2)).fill(1)
+    sell_pattern= Array(Math.round(time_cool_1c*max_temp_adjustment*2)).fill(1)
```

Comment:
Please define those variables before you use them.
You may insert `"use strict";` as the first line in the file to help you find identifiers that are not declared.